src/pages/docs/app-definition/fulfillment-delegation.mdx

```diff
@@ -0,0 +1,139 @@
+    </Description>
+    </Field>
+
+    <Field name="accountModals.registrationFormSchema" type="object" nullable={false} required={true}>
+    <Description>
+      [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/) that defines fields for registration.
+    </Description>
+    </Field>
+    <Field name="accountModals.registrationFormSchema.jsonSchema" type="object" nullable={false} required={true}>
+    <Description>
+      Defines the schema of your [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/). This will define the shape of the data that is being returned.
+    </Description>
+    </Field>
+    <Field name="accountModals.registrationFormSchema.uiSchema" type="object" nullable={false} required={true}>
+    <Description>
+      Defines the ui schema of your [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/)
+    </Description>
+    </Field>
+
+    <Field name="accountModals.settingsFormSchema" type="object" nullable={false} required={true}>
+    <Description>
+      [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/) that defines fields for changes to the connection
+    </Description>
+    </Field>
+    <Field name="accountModals.settingsFormSchema.jsonSchema" type="object" nullable={false} required={true}>
+    <Description>
+      Defines the schema of your [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/). This will define the shape of the data that is being returned.
+    </Description>
+    </Field>
+    <Field name="accountModals.settingsFormSchema.uiSchema" type="object" nullable={false} required={true}>
+    <Description>
+      Defines the ui schema of your [React JSON Schema Form](https://react-jsonschema-form.readthedocs.io/en/latest/)
+    </Description>
+    </Field>
+    
+    <Field name="images" type="object" nullable={false} required={true}>
+    <Description>
+      Images that will be used for this branded order source
+    </Description>
+    </Field>
+
+    <Field name="images.logoURL" type="string" nullable={false} required={true}>
+    <Description>
+      The full path to the logo used in modals and other areas of our platform for this order source. Use join(__dirname, '../assets/logo.svg') "/dev/integration/assets/ordersource1/logo.svg"
+    </Description>
+    </Field>
+
+    <Field name="images.iconUrl" type="string" nullable={false} required={true}>
+    <Description>
+      The full path to the icon used for this order source. Use join(__dirname, '../assets/logo.svg')  "/dev/integration/assets/ordersource1/icon.svg"
+    </Description>
+    </Field>
+
+</Reference>
+
+## Fulfillment Service
+
+<Reference>
+    <Field name="name" type="string" nullable={false} required={true}>
+    <Description>
+        Name of the shipping service
+    </Description>
+    </Field>
+
+    <Field name="abbreviation" type="object" nullable={true} required={false}>
```

Comment:
Is this an object? It has no documented fields.

Reply:
no it was a string. fixed it.